bindings/ergo-lib-wasm/src/wallet/derivation_path.rs

```diff
@@ -41,6 +41,30 @@ impl DerivationPath {
         )))
     }
 
+    /// Create root derivation path
+    pub fn master_path() -> Self {
+        DerivationPath(InnerDerivationPath::master_path())
+    }
+
+    /// Returns the length of the derivation path
+    pub fn depth(&self) -> usize {
+        self.0.depth()
+    }
+
+    /// Returns a new path with the last element of the deriviation path being increased, e.g. m/1/2 -> m/1/3
+    /// Returns an empty path error if the path is empty (master node)
+    pub fn next(&self) -> Result<DerivationPath, JsValue> {
+        Ok(self.0.next().map_err(to_js)?.into())
+    }
+
+    /// String representation of derivation path
+    /// E.g m/44'/429'/0'/0/1
+    #[wasm_bindgen(js_name = toString)]
+    #[allow(clippy::inherent_to_string)]
```

Comment:
I thought this was a better option for the `wasm` bindings, `toString` is used for displays in JS consoles/debuggers/etc, if I implemented `Display` trait we wouldn't get nice displays in JS land by default

Reply:
Good call!